### PR TITLE
APDS-85 - [BE] move permanent token authentication to a separate library

### PR DESCRIPTION
**Pull request by poxip**

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Change Log
+All notable changes to this project will be documented in this file.
```
Comment (remik) [inline CHANGELOG.txt]: Add 0.1 with initial version

Comment (poxip) [inline CHANGELOG.txt]: Okay, I will change it to 1.0.0 (as we talked about it, we will use semver).

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+When using the regular JWT login or the device logout view, use the ``X-Device-Model`` header to pass device model
+(otherwise, user agent will used instead as the name). After a successful login, the permanent token and id of the
+created device will be returned, for example:
+::
+
+  {
+      "token": "ads344fdgfd5454yJ0eAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VynRlYW1AYXJhYmVsLmxh",
+      "permanent_token": "gfd5454yJ0eAiOiJKV1QiLCJhbGciOiJ",
+      "device_id": 1
```
Comment (poxip) [inline README.rst]: This was agreed with @jacoor  in the previous pull requests (in https://github.com/ArabellaTech/django-rest-framework-jwt), permanent_token is too fragile.

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+To refresh JWT token, you have to pass the ``Permanent-Token`` header along with the request to identify the device.
+On success, response will return new JWT token (the same as it does after login).
+
+In case the permanent token has expired, the device will be logged out, and it will require login in again to obtain a
+new permanent token. To customize the expiration time and expiration accuracy, set the following settings in your
+``REST_FRAMEWORK`` configuration in **settings.py**
+
+
+**PermitHeaders middleware**
```
Comment (poxip) [inline README.rst]: Hmm, why not, will change.

**Comment by poxip** [comment]
@pkrzyzaniak Both of those things are implemented. You can list all user's devices (DeviceViewSet), and user can logout each device they are logged in.

**Pull request merged**